src/Compilers/VisualBasic/Portable/Parser/ParserFeature.vb

```diff
@@ -84,7 +85,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
 
                 Case Feature.LeadingDigitSeparator,
                     Feature.NonTrailingNamedArguments,
-                    Feature.PrivateProtected
+                    Feature.PrivateProtected,
+                    Feature.PROTOTYPECommnetsAfterLineContinuation
```

Comment:
Add `' PROTOTYPE(continuation-comments): Move the feature under VB16 once the LanguageVersion is available`